dvc/commands/experiments/__init__.py

```diff
@@ -58,6 +58,16 @@ def add_parser(subparsers, parent_parser):
     hide_subparsers_from_help(experiments_subparsers)
 
 
+def add_keep_selection_flag(experiments_subcmd_parser):
+    experiments_subcmd_parser.add_argument(
+        "--keep",
+        action="store_true",
+        default=False,
+        help="Keep the selected (committed, not queued) experiments "
```

Comment:
just the last thing to iterate / improve (I still don't quite satisfied with the message :) )

`committed` can be confusing (we use sometimes "commit an experiment" when we refer to the making it an actual commit (vs invisible in the `exp` namespace in Git)


how about "Keep the selected experiments instead of removing them"

For the `--queue` - I wonder if we should just raise an exception for now? Just don't allow that combination and put an explicit message - otherwise it's a bit confusing for people (--queue --keep still drops the experiments). WDYT?

Reply:
> For the --queue - I wonder if we should just raise an exception for now? Just don't allow that combination and put an explicit message - otherwise it's a bit confusing for people (--queue --keep still drops the experiments). 

Yep, that seems like a pretty good solution to keep things clear for everybody. That exception can be removed later  if we manage to find a nice way to make `--queue` work with other arguments (or deprecate it)
I filed issue [#10639](https://github.com/iterative/dvc/issues/10639) to discuss this in more detail.

> how about "Keep the selected experiments instead of removing them"
With the exception raised, we can go back to this simpler message. 

I'll push this fix asap.